2024-11-06 14:44:54,826 | WARNING | Неверный тип данных для объекта Runner
Traceback (most recent call last):
  File "E:\PYTHON\Proect_University\module_12\home_work_12_4\tests_12_4.py", line 82, in test_run
    obj2 = Runner(777, 10)
           ^^^^^^^^^^^^^^^
  File "E:\PYTHON\Proect_University\module_12\home_work_12_4\tests_12_4.py", line 12, in __init__
    raise TypeError(f'Имя может быть только строкой '
TypeError: Имя может быть только строкой int
2024-11-06 14:44:54,827 | WARNING | Неверная скорость для Runner
Traceback (most recent call last):
  File "E:\PYTHON\Proect_University\module_12\home_work_12_4\tests_12_4.py", line 69, in test_walk
    obj1 = Runner('Вася', -10)
           ^^^^^^^^^^^^^^^^^^^
  File "E:\PYTHON\Proect_University\module_12\home_work_12_4\tests_12_4.py", line 18, in __init__
    raise ValueError(f'Скорость не может быть отрицательной,'
ValueError: Скорость не может быть отрицательной, сейчас -10
